plugins/UsersManager/API.php

```diff
@@ -722,10 +889,12 @@ public function deleteUser($userLogin)
         Piwik::checkUserHasSuperUserAccess();
```

Comment:
@mattab after letting admin users add user to a site, I would say we also need to let them delete users if they have access to only the same sites (or less) as the admin user? Otherwise the admin only has the option to remove access to any site, but never be able to delete the user...

Reply:
Sounds good indeed, could you make this improvement @diosmosis ?

Reply:
If we show a delete icon next to some users and not next to others I think it would be confusing. Ok to add a modal like the "add existing user" modal that uses a username/email?

Reply:
The modal should work :+1: